src/nbl/asset/utils/waveContext.h

```diff
@@ -103,24 +109,27 @@ struct preprocessing_hooks final : public boost::wave::context_policies::default
             return true;
         }
         
-        if (strcmp(optionStr, "dxc_compile_flags") == 0 && hash_token_occurences == 1) {
+        if (strcmp(optionStr, "dxc_compile_flags") ) {
+            if (0 && hash_token_occurences != 1) {
```

Comment:
some weird typo here, you go into the if block if your optionStr DOES NOT equal `dxc_compile_flags`